src/common_utils/language_utils/sentence_filter_utils.py

```diff
@@ -1,4 +1,4 @@
-import src.common_utils.constants as constants
+import configuration as configuration
```

Comment:
#CleanCode
As I know 'import configuration' would have the same effect as 'import configuration as configuration' because you are not changing the name after 'as'